packages/components/src/internal/app/constants.ts

```diff
@@ -147,6 +154,19 @@ export const BIOLOGICS_APP_PROPERTIES: AppProperties = {
     baseProductHelpLinkPrefix: BASE_APP_HELP_LINK,
 };
 
+export const LIMS_APP_PROPERTIES: AppProperties = {
+    productId: LIMS_PRODUCT_ID,
+    name: LIMS_PRODUCT_NAME,
+    logoWithTextImageUrl: imageURL('lims/images', 'LK-LIMS-appmenu-WHITE.svg'),
+    logoBadgeImageUrl: imageURL('lims/images', 'LK-LIMS-Badge-WHITE.svg'),
+    logoBadgeColorImageUrl: imageURL('lims/images', 'LK-LIMS-Badge-COLOR-LIGHT.svg'),
+    controllerName: LIMS_CONTROLLER_NAME,
+    moduleName: 'sampleManagement',
+    searchPlaceholder: SAMPLE_MANAGER_SEARCH_PLACEHOLDER,
+    dataClassUrlPart: SOURCES_KEY,
```

Comment:
These two seem a bit odd to be in the AppProperties. I think we have other app context places that would likely make more sense for these...but that is for another PR

Reply:
Yes, we can rationalize this in another PR.